src/ose.js

```diff
@@ -75,11 +79,11 @@ Hooks.once("init", async () => {
   CONFIG.Actor.documentClass = OseActor;
   CONFIG.Item.documentClass = OseItem;
 
-  CONFIG.Actor.systemDataModels = {
```

Comment:
Is this going to break things for people on v10 or v11? Same with the next set of changes.

Reply:
Reverted, I didn't know that we were keeping v10 support